hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/RemoteException.java

```diff
@@ -47,12 +50,23 @@ public RemoteException(String className, String msg) {
    * @param erCode may be null
    */
   public RemoteException(String className, String msg, RpcErrorCodeProto erCode) {
+    this(className, msg, erCode, null);
+  }
+
+  /**
+   * @param className wrapped exception, may be null
```

Comment:
nit: add a . here and in other javadocs, to keep the compilers happy

Reply:
Updated.